w_common/lib/src/common/managed_stream_subscription.dart

```diff
@@ -54,16 +54,16 @@ class ManagedStreamSubscription<T> implements StreamSubscription<T> {
   }
 
   @override
-  void onData(void handleData(T _)) => _subscription.onData(handleData);
+  void onData(void handleData(T _)?) => _subscription.onData(handleData);
 
   @override
-  void onDone(void handleDone()) => _wrapOnDone(handleDone);
+  void onDone(void handleDone()?) => _wrapOnDone(handleDone);
 
   @override
-  void onError(Function handleError) => _wrapOnError(handleError);
+  void onError(Function? handleError) => _wrapOnError(handleError);
 
   @override
-  void pause([Future<dynamic> resumeSignal]) =>
+  void pause([Future<dynamic>? resumeSignal]) =>
```

Comment:
Let's make this `Future<void>?` to match the `StreamSubscription` interface. Should be backwards-compat.